Assets/static/scripts/messenger.js

```diff
@@ -52,6 +52,11 @@ function message_to_html(message, displayName) {
 function push_message(messages, i) {
     if (i == messages.length) return;
 
+    if (!VISIBLE) {
+        document.getElementById("icon").href = "/images?path=favicon_notification.png";
+        new Audio("http://127.0.0.1:8080/sounds?path=notification.mp3").play();
```

Comment:
Can the URL here be `/sounds?path=notification.mp3`? Or does it have to include the hostname as well?

Reply:
Ah, my mistake! No, the hostname is not required there! Thats what I get for making a pull request at 3am 😅

Reply:
Haha no worries lol. I forgot that you could even omit the host name so 😂😂